code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -220,120 +221,52 @@
 
 /obj/item/storage/fancy/cigarettes/acmeco
 	name = "\improper AcmeCo cigarette packet"
-	desc = "A packet of six AcmeCo cigarettes. For those who somehow want to obtain the record for the most amount of cancerous tumors."
+	desc = "For those who somehow want to obtain the record for the most amount of cancerous tumors."
 	icon_state = "Apacket"
 	item_state = "Apacket"
 	cigarette_to_spawn = /obj/item/clothing/mask/smokable/cigarette/acmeco
 
 /obj/item/storage/fancy/cigarettes/blank
 	name = "\improper blank cigarette packet"
-	desc = "A packet of six blank cigarettes. The healthiest cigarettes on the market!"
+	desc = "The healthiest cigarettes on the market! Wait, isn't this just a roll of paper?"
 	icon_state = "Bpacket"
 	item_state = "Bpacket"
 	cigarette_to_spawn = /obj/item/clothing/mask/smokable/cigarette/blank
 
-/obj/item/storage/fancy/cigarettes/dromedaryco
-	name = "\improper DromedaryCo cigarette packet"
-	desc = "A packet of six imported DromedaryCo cancer sticks. A label on the packaging reads, \"Wouldn't a slow death make a change?\""
-	icon_state = "Dpacket"
-	item_state = "Dpacket"
-	cigarette_to_spawn = /obj/item/clothing/mask/smokable/cigarette/dromedaryco
-
-/obj/item/storage/fancy/cigar
+/obj/item/storage/fancy/cigarettes/cigar
 	name = "cigar case"
-	desc = "A case for holding your cigars when you are not smoking them."
+	desc = "A luxurious tote for your fat tokes."
```

Comment:
Cringe

Reply:
it rhymes, also _marginally_ better than
name : "cigar case"
desc : "case for holding cigars" which is basically the same thing but more verbose